aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move

```diff
@@ -85,8 +106,14 @@ module post_mint_reveal_nft::minting {
         property_types: vector<String>,
     }
 
-    /// WhitelistMintConfig stores information about whitelist minting.
+    /// WhitelistMintConfig stores information about all stages of whitelist.
+    /// Most whitelists are one-stage, but we allow multiple stages to be added in case there are multiple rounds of whitelists.
     struct WhitelistMintConfig has key {
+        whitelist_configs: vector<WhitelistMintConfigSingleStage>,
+    }
+
+    /// WhitelistMintConfigSingleStage stores information about one stage of whitelist.
+    struct WhitelistMintConfigSingleStage has store {
```

Comment:
Can we just name this whitelist config? WhitelistMintConfigSingleStage sounds both confusing and complicated.